src/routes/+page.svelte

```diff
@@ -29,30 +29,32 @@
 	];
 </script>
 
+{#if showSponsors}
+	<Sponsors />
+{/if}
+
 <div class="introduction primary-bg">
 	<div class="section">
-		<p class="text-center">
-			DDD South West is an inclusive, non-profit developer conference organised by the community,
-			for the community.
+		<h2>Latest updates</h2>
+		<p>
+			<a href="/schedule">Schedule is announced!!</a> Time for last second tweaks to your talks, test
+			your demos and get ready for the big day!!
```

Comment:
On larger screen resolutions, this text is bigger than the associated `h2` which looks odd. Perhaps we could shrink it a little?